src/translation/translation_pt_PT.ts

```diff
@@ -3814,13 +3814,13 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
     <message>
         <location filename="../serverdlg.cpp" line="175"/>
         <source>Display dialog to select recording directory button</source>
-        <translation>Mostrar diálogo para selecionar botão do directorio de gravação</translation>
+        <translation>Exibir janela para selecionar o botão da pasta de gravação</translation>
```

Comment:
Also needs to be potentially reverted.
@mansil please comment

Reply:
I reverted it. @mansil doesn't seem to be here. See also my comment on Discord.